observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,72 @@ These release notes track the development of the Network Observability Operator
+// commenting until CLI feature doc is merged 
+// xref:../../observability/network_observability/netobserv_cli/netobserv-cli-install.adoc#network-observability-netoberv-cli-about_netobserv-cli-install[Network Observability CLI]
+
+[id="network-observability-operator-1.6.0-bug-fixes"]
+=== Bug fixes
+* Previously, a dead link to the OpenShift documentation was displayed in the OLM form for `FlowMetrics` API creation. Now the link has been updated to point to a valid page. (link:https://issues.redhat.com/browse/NETOBSERV-1607[*NETOBSERV-1607*])
+* Previously, the Operator description in Operator Hub was displaying a broken link to the documentation. With this fix, this link is restored. (link:https://issues.redhat.com/browse/NETOBSERV-1544[*NETOBSERV-1544*])
+* Previously, if Loki was disabled and the Loki `Mode` was set to `LokiStack`, or if Loki manual TLS configuration was configured, the Operator still tried to read the Loki CA certificates. With this fix, when Loki is disabled, the Loki certificates are not read, even if there are settings in the Loki configuration. (link:https://issues.redhat.com/browse/NETOBSERV-1647[*NETOBSERV-1647*])
+* Previously, the must-gather plugin for the Network Observability Operator was only working on `amd64` architectures and failing on all others because the plugin was using amd64 for `oc` binary. Now, the Network Observability Operator must-gather plugin collects logs on any architecture platform. (link:https://issues.redhat.com/browse/NETOBSERV-1618[*NETOBSERV-1618*])
+* Previously, when filtering on IP addresses using `not equal to`, the Operator would return a request error.
+Now, the IP filtering works in both `equal` and `not equal to` cases for IPs addresses and ranges. (link:https://issues.redhat.com/browse/NETOBSERV-1630[*NETOBSERV-1630*])
+* Previously, when a user was not an admin, the error messages were not consistent with the selected tab of the *Network Traffic* view in the web console. Now, the `user not admin` error displays on any tab with improved display.(link:https://issues.redhat.com/browse/NETOBSERV-1621[*NETOBSERV-1621*])
+
+
+[id="network-observability-operator-1.6.0-known-issues"]
+=== Known issues
+* When the eBPF agent `PacketDrop` feature is enabled, and sampling is configured greater than 1, reported dropped bytes and packet ignore the sampling configuration. While this is done on purpose to not miss any drop, a side effect is that the reported proportion of drops versus non-drops becomes biased. At very high sampling rate, such as 1:1000, it's likely that almost all the traffic appears to be dropped, when looked from the console plugin. But this is only a bias. (link:https://issues.redhat.com/browse/NETOBSERV-1676[*NETOBSERV-1676*])
+* In the "Manage panels" dialog in the Overview tab, filtering on "total", "bar", "donut", or "line" doesn't show any result. (link:https://issues.redhat.com/browse/NETOBSERV-1540[*NETOBSERV-1540*])
```

Comment:
```suggestion
* In the "Manage panels" dialog in the Overview tab, filtering on "total", "bar", "donut", or "line" doesn't show any result. (link:https://issues.redhat.com/browse/NETOBSERV-1540[*NETOBSERV-1540*])
```
Wonder if the quotes are supposed to be bold because they are GUI interface references?

Reply:
Yes I think so. I bolded them.